<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InformationDialog</class>
 <widget class="QDialog" name="InformationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1179</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>INFORMATION PROPERTY</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>1171</width>
     <height>660</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="labelModelName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>105</y>
      <width>391</width>
      <height>86</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Model Name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonClose">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>551</y>
      <width>200</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelVersion">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>391</width>
      <height>86</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Version</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelHddSize">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>275</y>
      <width>391</width>
      <height>86</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>SSD Size</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelGps">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>445</y>
      <width>391</width>
      <height>86</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>GPS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelLicensePlate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>391</width>
      <height>86</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>License Plate</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelLicensePlate2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>740</width>
      <height>86</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>EASTERN</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelModelName2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>105</y>
      <width>740</width>
      <height>86</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>EMS-S10</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelVersion2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>190</y>
      <width>740</width>
      <height>86</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>0.0.0 2016.12.30</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNormalSize2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>275</y>
      <width>371</width>
      <height>86</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>888 GigaByte</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelEventSize2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>275</y>
      <width>370</width>
      <height>86</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>555 GigaByte</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelGps2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>445</y>
      <width>740</width>
      <height>86</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Not Connected</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDiskTemperature">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>360</y>
      <width>370</width>
      <height>86</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>SSD : 999 ℃</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDVRTemperature2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>360</y>
      <width>371</width>
      <height>86</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>DVR : 999 ℃</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDVRTemperature">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>391</width>
      <height>86</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Temperature</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
